src/test/resources/output/json/00196/TT001966831.json

```diff
@@ -37,13 +37,6 @@
   } ],
   "hbzId" : "TT001966831",
   "id" : "http://lobid.org/resources/TT001966831#!",
-  "isPartOf" : [ {
-    "id" : "http://lobid.org/resources/HT006987933#!",
-    "label" : "lobid Ressource"
-  }, {
-    "id" : "http://ld.zdb-services.de/resource/626580-7",
-    "label" : "http://ld.zdb-services.de/resource"
-  } ],
```

Comment:
Here we loose the connection to ld.zdb-services.de/resource/626580-7 whih is not redundantly covered by another field. Is this intended?